schematic/utils/csv_utils.py

```diff
@@ -187,7 +191,7 @@ def check_schema_definition(schema_definition: pd.DataFrame) -> bool:
         "Requires" in list(schema_definition.columns) or
         "Requires Component" in list(schema_definition.columns)
     ):
-        print(
+        logger.error(
```

Comment:
Look for all instances of `logger.error()` (other than the Google one), and replace with Exceptions.